.github/workflows/enos-ci-cleanup.yml

```diff
@@ -0,0 +1,92 @@
+name: enos-ci-cleanup
```

Comment:
Can you rename this and the file to: `test-ci-cleanup`. Ryan asked that I do the same in my switch over PR.

Reply:
yep!